cloudflare/resource_cloudflare_fallback_domain.go

```diff
@@ -3,11 +3,20 @@ package cloudflare
 import (
 	"context"
 	"fmt"
+	"log"
 
 	cloudflare "github.com/cloudflare/cloudflare-go"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+// use this since we cannot use const for lists.
+func default_domains() []string {
+	return []string{
```

Comment:
@kjhandy i can see internally we have this stored; is it something we plan on exposing via the public API to save storing this in Terraform?

Reply:
I saw the teams api endpoint (api.teams.cloudflare.com) allowed `&reset_defaults=true` during a `DELETE` call, but the public API doesn't allow this.  So, I opted to just duplicate it here.

Reply:
i'll chat to some folks and see what the intention is there. that would make more sense to get publicly supported so the service is the source of truth for this list instead of Terraform.

Reply:
cool, confirmed with the team this is 👌 if we add it to cloudflare-go, we can do away with this list entirely.